### PR TITLE
Fix error on adding reaction

### LINKED ISSUE #778: [Android] Error on adding reaction
Got an error while adding reaction.

screen shot 2017-07-01 at 2 58 24 pm

### PR DESCRIPTION
**Pull request by saketkumar95**
This fixes #778

### CODE REVIEW

**Comment by borisyankov** [comment]
Can you explain what the issue was and why this is fixing it?

**Comment by saketkumar95** [comment]
there was a error when adding tapping on the reactions. The error thrown : 
screen shot 2017-07-01 at 2 58 24 pm


Probably what i got the cause was , Having numbers in `<Text>` so I converted it to string and the error was gone.

**Comment by saketkumar95** [comment]
@borisyankov You can reproduce it just by tapping or adding reactions on Android 4.3 😄

**Comment by borisyankov** [comment]
But why adding an empty string fixes the issue?

**Comment by saketkumar95** [comment]
@borisyankov No I removed that empty string now.

**Comment by borisyankov** [comment]
I see you are converting the number to a string in a different way.
Still, the question is: what was the issue? Why is this a fix for that issue? Will the issue materialize everywhere we are using numbers inside a `<Text />`?

**Comment by saketkumar95** [comment]
yeah maybe we want to materialize  everywhere we are using numbers inside a `<Text />` 
For more details checkout this https://github.com/facebook/react-native/issues/12905

**Comment by borisyankov** [comment]
This bug is specific to RN 0.43.x
It has been fixed in later versions.
I don't think doing these manual fixes make much sense.

**Comment by saketkumar95** [comment]
But currently this occurs. So what should we do to avoid?

**Comment by borisyankov** [comment]
We need to upgrade React Native.
Until then, using RawLabel here is probably a solution.

**Comment by saketkumar95** [comment]
RawLabel uses `<Text />` and this issue is for text only? How this is a solution?

**Comment by borisyankov** [comment]
Then you can convert to string in a single place and remove when upgraded to RN 0.44

**Comment by saketkumar95** [comment]
okay so I need to materialize `<Text />` everywhere we are using numbers? I thought of adding only here. 😄

**Comment by saketkumar95** [comment]
@borisyankov Updated! Will update with RawLabel on other places in another PR.

File: src/common/RawLabel.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default class RawLabel extends PureComponent {
 
     return (
       <Text style={[styles.label, style]} {...restProps}>
-        {text}
+        {`${text}`}
```
Comment (borisyankov) [inline src/common/RawLabel.js]: Make this more explicit. 
`{text.toString()}` shows intent better.

Comment (saketkumar95) [inline src/common/RawLabel.js]: Done! 👍

**Comment by saketkumar95** [comment]
@borisyankov I don't get what's going wrong in checks on github. But One question, I see `RawLabel` are used many places. I think it should affect some places where it's working fine right now.

**Pull request closed without merging**